PopupBridge/src/main/java/com/braintreepayments/api/internal/AnalyticsApi.kt

```diff
@@ -0,0 +1,84 @@
+package com.braintreepayments.api.internal
+
+import android.content.Context
+import com.braintreepayments.api.network.PostRequestExecutor
+import org.json.JSONArray
+import org.json.JSONObject
+import java.net.URL
+
+internal class AnalyticsApi(
+    context: Context,
+    private val postRequestExecutor: PostRequestExecutor = PostRequestExecutor(),
+    private val deviceRepository: DeviceRepository = DeviceRepository(context),
+    private val analyticsParamRepository: AnalyticsParamRepository = AnalyticsParamRepository.instance,
+    private val time: Time = Time(),
+) {
+
+    suspend fun execute(eventName: String) {
+        postRequestExecutor.execute(
+            url = URL(FPTI_URL),
+            jsonBody = createJsonBody(eventName).toString()
+        )
+    }
+
+    private fun createJsonBody(eventName: String): JSONObject {
+        val batchParams = JSONObject().apply {
+            put(KEY_APP_ID, deviceRepository.getAppId())
+            put(KEY_APP_NAME, deviceRepository.getAppName())
+            put(KEY_CLIENT_OS, deviceRepository.androidApiVersion)
+            put(KEY_CLIENT_SDK_VERSION, deviceRepository.sdkVersion)
+            put(KEY_COMPONENT, "popupbridgesdk")
+            put(KEY_DEVICE_MANUFACTURER, deviceRepository.deviceManufacturer)
+            put(KEY_DEVICE_MODEL, deviceRepository.deviceModel)
+            put(KEY_EVENT_SOURCE, "mobile-native")
+            put(KEY_IS_SIMULATOR, deviceRepository.isDeviceEmulator)
+            put(KEY_MERCHANT_APP_VERSION, deviceRepository.getAppVersion())
+            put(KEY_PLATFORM, "Android")
+            put(KEY_SESSION_ID, analyticsParamRepository.sessionId)
+            put(KEY_TENANT_NAME, "Braintree")
+        }
+
+        val eventParams = JSONArray().apply {
+            put(JSONObject().apply {
+                put(KEY_EVENT_NAME, eventName)
+                put(KEY_TIMESTAMP, time.currentTime)
+
+            })
+        }
+
+        val events = JSONArray().apply {
+            put(JSONObject().apply {
+                put("batch_params", batchParams) // No constant defined for this key
+                put(KEY_EVENT_PARAMETERS, eventParams)
+            })
+        }
+
+        return JSONObject().apply {
+            put(KEY_EVENTS, events)
+        }
+    }
+
+    companion object {
+        private const val FPTI_URL = "https://api-m.paypal.com/v1/tracking/batch/events"
```

Comment:
On iOS we removed to `-m` a while ago (this PR: https://github.com/braintree/braintree_ios/pull/1175) not sure why we didn't align that on Android but this may be a good opportunity to do that.

Reply:
Thanks for calling this out! I removed the `-m` and retested the API call.